README.md

```diff
@@ -67,3 +67,4 @@ List of every HackerOoT contributors, from most recent to oldest contribution:
 - ariahiro64
 - ghost
 - krm01
+- Reonu
```

Comment:
you should move your name at the top of the list since it's ordered from recent to oldest, alternatively you can reverse the order of that list

Reply:
dont reverse it pls i wanna be at the bottom